schedule/yast/autoyast_salt_formulas.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: autoyast_salt_formulas
+description: |
+  Test installation using AY plus salt formulas.
```

Comment:
AY is not official abbreviation, we use it sometimes for file names, but for description I would go for `AutoYaST`. Besides it would be nice to add some info, what you provision, motd, etc, kind of what you wrote in the variables.pm.

Reply:
Ay to AutoYaST changed. But the general description of the suite is adequate in my opinion with the important stuff. More info might also make it bound to the test changes